plugins/modules/uptime_check.py

```diff
@@ -21,31 +21,59 @@
     apikey:
         required: true
         description:
-            - api key to auth with Pingdom
-    url:
+            - The user's API key used to authorize the log in into Pingdom is taken as a string.

```

Comment:
Please can you add the option type to each option, see https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#documentation-fields

Reply:
Added "type" options to all attributes.